code/src/App.js

```diff
@@ -1,9 +1,84 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import { HAPPY_THOUGHTS_URL, LIKE_THOUGHT_URL } from './reusable/urls'
+
+import SendThought from './components/SendThought'
+import ThoughtsList from './components/ThoughtsList'
+import ErrorPopup from './components/ErrorPopup'
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList] = useState([])
+  const [newThought, setNewThought] = useState('')
+  const [error, setError] = useState(false)
+  
+  useEffect(() => {
+    fetchThoughtsList()
+  }, [])
+
+  //GET
+  const fetchThoughtsList = () => {
+    fetch(HAPPY_THOUGHTS_URL)
+      .then(response => response.json())
+      .then(thoughts => setThoughtsList(thoughts))
+      .catch(err => console.error(err))
```

Comment:
console outputs can't be present in the production code.

Reply:
Do you mean to remove the whole catch() or to replace the console.error(err) to show something like <> </>? 
What would be best practice? :)